zulip_bots/zulip_bots/bots/lunch/doc.md

```diff
@@ -0,0 +1,34 @@
+# Lunch Bot 🍛
+A small Zulip bot for organizing lunch with your friends and coworkers.
+
+This bot is a Python clone of the [Hubot-Lunchbot](
+https://www.npmjs.com/package/hubot-lunchbot) so if you are familiar with that
+bot you'll likely feel right home with this one.
+
+## How does it work
+Each user is allowed to create a type of shop list for meals
+
+## Commands
+`<user|optional> (I) want(s) <item>`
+Adds <item> to the sender's (or <user>'s if specified) list of items
+
+`<user> I want that`
+Adds every item in <user>'s list of items to the sender's list of items
+
+`<user|optional> remove <item>`
+Removes <item> to the sender's (or <user>'s if specified) list of items
+
+`<user|optional> remove all`
+Removes all items to the sender's (or <user>'s if specified) list of items
+
+`orders`
+Show all orders
+
+`<user|optional> order`
+Shows all items of the sender's (or <user>'s if specified) list of items
+
+`who should <verb>(?)`
+Randomly selects a user from the orders list
+
+`help`
+Shows help (this section)
```

Comment:
Could you please modify the `doc.md` to follow a structure similar to [the xkcd bot's docs](https://github.com/zulip/python-zulip-api/blob/master/zulip_bots/zulip_bots/bots/xkcd/doc.md)? We're hoping that's the structure we can follow moving further! :)